mitreattack/navlayers/generators/sum_generator.py

```diff
@@ -6,14 +6,14 @@
 
 
 class BatchGenerator:
-    def __init__(self, source, domain='enterprise', local=None):
+    def __init__(self, source, domain='enterprise', resource=None):
         """
         Initialize the Generator
-        :param source: Which source to use for data (local or taxii [server])
+        :param source: Which source to use for data (local, taxii [server], or [remote] ATT&CK Workbench)
         :param domain: Which matrix to use during generation
-        :param local: Optional path to local data
+        :param resource: string path to local cache of stix data (local) or url of workbench to reach out to (remote)
```

Comment:
I'm concerned that the word "cache" might confuse people here. Probably just remove the words "cache of" and just let it be "path to local STIX data"

Reply:
Works for me.